provider/datagen/tests/data/json/units/constants@1/und.json

```diff
@@ -1,19 +1,228 @@
+        24
+      ],
+      "denominator": [
+        96,
+        242,
+        206,
+        7
+      ],
+      "sign": "Positive",
+      "constant_type": "Approximate"
+    },
+    "ft2_to_m2": {
+      "numerator": [
+        9,
+        55,
+        2
+      ],
+      "denominator": [
+        132,
+        215,
+        23
+      ],
+      "sign": "Positive",
+      "constant_type": "Actual"
+    },
+    "ft3_to_m3": {
+      "numerator": [
+        101,
+        232,
+        75,
+        3
+      ],
+      "denominator": [
+        136,
+        82,
+        106,
+        116
+      ],
+      "sign": "Positive",
+      "constant_type": "Actual"
+    },
+    "ft_to_m": {
+      "numerator": [
+        125,
+        1
+      ],
+      "denominator": [
+        226,
+        4
+      ],
+      "sign": "Positive",
+      "constant_type": "Actual"
+    },
+    "gal_imp_to_m3": {
+      "numerator": [
+        209,
+        239,
+        6
+      ],
+      "denominator": [
+        0,
+        225,
+        245,
+        5
+      ],
+      "sign": "Positive",
+      "constant_type": "Actual"
+    },
+    "gal_to_m3": {
+      "numerator": [
+        153,
+        25,
+        52,
+        28
+      ],
+      "denominator": [
+        0,
+        162,
+        148,
+        26,
+        29
+      ],
+      "sign": "Positive",
+      "constant_type": "Actual"
+    },
+    "glucose_molar_mass": {
+      "numerator": [
+        85,
+        125,
+        27
+      ],
+      "denominator": [
+        16,
+        39
+      ],
+      "sign": "Positive",
+      "constant_type": "Actual"
+    },
+    "gravity": {
+      "numerator": [
+        37,
+        254,
+        2
+      ],
+      "denominator": [
+        32,
+        78
+      ],
+      "sign": "Positive",
+      "constant_type": "Actual"
+    },
+    "in3_to_m3": {
+      "numerator": [
+        127,
+        65,
+        31
+      ],
+      "denominator": [
+        0,
+        162,
+        148,
+        26,
+        29
+      ],
+      "sign": "Positive",
+      "constant_type": "Actual"
+    },
+    "item_per_mole": {
+      "numerator": [
+        0,
+        0,
+        198,
+        92,
+        20,
+        95,
+        41,
+        23,
+        134,
+        127
+      ],
+      "denominator": [
+        1
+      ],
+      "sign": "Positive",
+      "constant_type": "Actual"
+    },
+    "lb_to_kg": {
+      "numerator": [
+        133,
+        32,
+        180,
+        2
+      ],
+      "denominator": [
+        0,
+        225,
+        245,
+        5
+      ],
+      "sign": "Positive",
+      "constant_type": "Actual"
+    },
+    "meters_per_AU": {
+      "numerator": [
+        108,
+        90,
+        186,
+        212,
+        34
+      ],
+      "denominator": [
+        1
+      ],
+      "sign": "Positive",
+      "constant_type": "Actual"
+    },
+    "sec_per_julian_year": {
+      "numerator": [
+        224,
+        135,
+        225,
+        1
+      ],
```

Comment:
Suggestion (optional): instead of using a plain ZeroVec you could make a newtype wrapper over ZeroVec that implements a human-readable JSON serialization and also includes functions for converting to/from BigInt

Reply:
This on is #4092

Reply:
https://github.com/unicode-org/icu4x/issues/4092